docs/interpreter/hive.md

```diff
@@ -124,6 +124,16 @@ Hive Interpreter will be deprecated and merged into JDBC Interpreter. You can us
     <td></td>
     <td><b>( Optional ) </b>Other properties used by the driver of <code>%hive(${prefix})</code> </td>
   </tr>
+  <tr>
+    <td>zeppelin.jdbc.auth.type</td>
+    <td></td>
+    <td><b>( Optional ) </b>If auth type is needed, Example: KERBEROS/SIMPLE</td>
+  </tr>
+  <tr>
+    <td>hive.proxy.user</td>
+    <td>true</td>
+    <td><b>( Optional ) </b>If want to use `hive.server2.proxy.user`</td>
```

Comment:
These back quoters won't work properly in `<table>` tag. So please use `<code>` tag like this:
`<code>hive.server2.proxy.user</code>`